code/API_definitions/BYON-Webrtc-Events.yaml

```
@@ -0,0 +1,1520 @@
+
+    ### 3.2. Registration events
+
+    Find more information at the Schema description.
+
+    A `org.camaraproject.webrtc-events.registration` event includes two main use cases: update about netowrk status of the registration **and** provide notification about requested sessions (incoming calls).
+
+    On one hand, it provides the same session information received using the WebRTC BYON RACM API, when retrieving information from the GET /racm/sessions/{racmSessionId} endpoint. Mainly it includes a unique registration ID, and the registration status. A valid `Registered` registration status grants that endpoint is visible for the network, since an invalid `Unregistered` status value means that network considers your endpoint unreachable.
+
+    When a new session is requested, it includes the context information: caller and possible carrier data to contextualize (call subject, branding info, etc) and a unique SessionId, with this SessionId is possible to create a subscription and receive events about the call itself and its status progressing on the telco network.
+
+    **NOTE**: The `Unregistered` status will be only provided by the notification service, a GET /racm/sessions/ for an unregistered endpoint will produce an 404 response.
+
+    ## 4. Authorization and authentication
+
+    The "Camara Security and Interoperability Profile" provides details on how a client requests
+    an access token. Please refer to Identify and Consent Management
+    (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of
+    the Profile.
+
+    Which specific authorization flows are to be used will be determined during onboarding process,
+    happening between the API Client and the API Provider, taking into account the declared purpose
+    for accessing the API, while also being subject to the prevailing legal framework dictated by
+    local legislation.
+
+    It is important to remark that in cases where personal user data is processed by the API, and
+    users can exercise their rights through mechanisms such as opt-in and/or opt-out, the use of
+    3-legged access tokens becomes mandatory. This measure ensures that the API remains in strict
+    compliance with user privacy preferences and regulatory obligations, upholding the principles
+    of transparency and user-centric data control.
+
+    ## 5. Further info and support
+    (Will be added in a later version of the documentation)
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
+  x-camara-commonalities: 0.4.0
+
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+servers:
+  - url: "{apiRoot}/webrtc-events/v1"
```

According to https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#53-api-versions-throughout-the-release-process
the `url` should end with `vwip`
```suggestion
  - url: "{apiRoot}/webrtc-events/vwip"
 ``` 
 When the release is prepared it should be changed to `{apiRoot}/webrtc-events/v0.1rc` and finally to `{apiRoot}/webrtc-events/v0.1`
 https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#52-api-version-in-url-oas-servers-object

---

Fixed at [93a4bcf](https://github.com/camaraproject/WebRTC/pull/53/commits/93a4bcf6f12c8fa4792f9be936cffe28c7e06d5b)